attributes/default.rb

```diff
@@ -131,13 +131,17 @@
 
 # NVIDIA
 default['cfncluster']['nvidia']['enabled'] = 'no'
-default['cfncluster']['nvidia']['driver_version'] = '460.73.01'
-default['cfncluster']['nvidia']['driver_url'] = 'https://us.download.nvidia.com/tesla/460.73.01/NVIDIA-Linux-x86_64-460.73.01.run'
-default['cfncluster']['nvidia']['cuda_version'] = '11.3'
-default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.download.nvidia.com/compute/cuda/11.3.0/local_installers/cuda_11.3.0_465.19.01_linux.run'
-
-# NVIDIA fabric-manager
-default['cfncluster']['nvidia']['fabricmanager']['package'] = "nvidia-fabricmanager-460"
+default['cfncluster']['nvidia']['driver_version'] = '470.82.01'
+default['cfncluster']['nvidia']['driver_url'] = 'https://us.download.nvidia.com/tesla/470.82.01/NVIDIA-Linux-x86_64-470.82.01.run'
+default['cfncluster']['nvidia']['cuda_version'] = '11.4'
+default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.download.nvidia.com/compute/cuda/11.4.3/local_installers/cuda_11.4.3_470.82.01_linux.run'
+
+# The package name of Fabric Manager for alinux2 and centos7 is nvidia-fabric-manager-version
+# For ubuntu, it is nvidia-fabricmanager-470_version
```

Comment:
For ubuntu is nvidia-fabricmanager-<version>

Reply:
It is for example: nvidia-fabricmanager-470_470.82.01, 
I can change the comment to nvidia-fabricmanager-majorversion_version